wikimedia/wikipedia-ios
# Add export user data flow

**Phabricator:** 
https://phabricator.wikimedia.org/T258212

### Notes
This feature allows users to export their app container via a `UIActivityViewController` from the Help and Feedback screen.

### Test Steps
1. Fresh install, log in, enable syncing via app Settings > Article storage and syncing.
2. Create some reading lists and save some articles.
3. Go to App Settings > Help and feedback.
4. Tap "Export User Data" button at the bottom
5. After a while, you should see a share activity sheet pop up. Airdrop the zip file to your machine.
6. Open the zip file, confirm you see data (console.log, Permanent Cache directory, User Data Export Sync Info json file, and some Wikipedia.sqlite database files).
